authorization/src/main/java/uk/gov/di/ipv/cri/common/api/handler/AuthorizationHandler.java

```diff
@@ -70,6 +72,12 @@ public APIGatewayProxyResponseEvent handleRequest(
             authorizationValidatorService.validate(authenticationRequest, sessionItem);
 
             // create authorization
+            if (sessionItem.getAuthorizationCode() == null
+                    || StringUtils.isBlank(sessionItem.getAuthorizationCode()))
+                sessionService.createAuthorizationCode(sessionItem);
+
+            eventProbe.log(INFO, "Authorization Code: " + sessionItem.getAuthorizationCode());
```

Comment:
I think @idrop suggestion was to only log that an authorization code has been assigned when it hasn't already been set.
Furthermore, the authorizationCode is sensitive data and should not be logged.
Therefore, change the eventProbe.log invocation to only happen when an authorization code is assigned and adjust the log message accordingly